app/modules/hub/hub-aura3d.js

```
@@ -0,0 +1,496 @@
+          if (
+            inner in CONFIG.boundaryElasticity &&
+            typeof value === 'number' &&
+            Number.isFinite(value)
+          ) {
+            CONFIG.boundaryElasticity[inner] = value;
+          }
+        });
+        return;
+      }
+      if (key === 'noiseIntensity') {
+        if (typeof value === 'number' && Number.isFinite(value)) {
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)